supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -259,6 +259,41 @@ When combined with privileges, permissions define a user's or resource's overall
 
 *See also*:
 
+[discrete]
+[[plugin-adj]]
+==== image:images/yes.png[yes] plugin (adjective)
+*Description*: Use to distinguish software code separate from the core application or service that adds new features or extends the functionality. 
+Use "plugin" rather than "plug-in", unless you are updating existing content that uses the hyphenated form.
+
+*Use it*: yes
+
+*Incorrect forms*: plug in, plug-in
+
+*See also*: xref:plugin[plugin]
+
+[discrete]
+[[plugin]]
+==== image:images/yes.png[yes] plugin (noun)
+*Description*: A "plugin" is a software component that adds new features or extends the functionality of an existing application or service.
```

Comment:
Sorry noticed one last thing. "plugin" should be in italics here, since we're defining the term

Reply:
@bergerhoffer - Thanks for flagging. Fixed!